### ISSUE TITLE
false positive with empty_parentheses_with_trailing_closure

### ISSUE THREAD

**Issue by BluMist** [realm/SwiftLint#1021]
@marcelofabri 

```swift
UIView.animateWithDuration(0.3, animations: {
    self.disableInteractionRightView.alpha = 0
}, completion: { _ in
    ()
})

**Comment by marcelofabri** [comment]
@BluMist I was able to reproduce it, thanks for reporting!

**Comment by marcelofabri** [comment]
@BluMist can you try https://github.com/realm/SwiftLint/pull/1022 to see if it solves the issue for you?

**Comment by BluMist** [comment]
confirming I can no longer produce false positives with #1022 locally
